src/services/expenses.service.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const { filterExpenses } = require('../helpers/filterExpenses');
+const { Expense } = require('../models/Expense');
+
+async function getAllExpenses(userId, categories, from, to) {
+  const expensesFromDB = await Expense.findAll();
+
+  if (!userId && !categories && !from && !to) {
+    return expensesFromDB;
+  }
+
+  try {
+    const filteredExpenses = filterExpenses(
+      expensesFromDB, categories, userId, from, to
+    );
+
+    return filteredExpenses;
+  } catch (error) {
+    return error;
+  }
+}
+
+async function getExpenseById(id) {
+  try {
+    const expense = await Expense.findByPk(id);
+
+    return expense;
+  } catch (error) {
+    return error;
+  }
```

Comment:
The same, try/catch isn't a good idea in your `services` you can just catch errors in the controllers and just send error info to the client